frontend/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -9,8 +11,9 @@ const LandingPage = () => (
   <>
     <Hero />
     <KeyFeaturesSection />
-    <InteractiveViewSection />
+    {inDev && <InteractiveViewSection />}
```

Comment:
making this as an in dev feature atm since it has references to graphical selector which is not ready to be released

Reply:
ye betto (whip)